packages/hub/db/migrations/20220413090421591_card-space-unused-data-cleanup.ts

```diff
@@ -0,0 +1,24 @@
+import { MigrationBuilder } from 'node-pg-migrate';
+
+const TABLE = 'card_spaces';
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.dropColumns(TABLE, [
+    'bio_description',
+    'bio_title',
+    'donation_description',
+    'donation_suggestion_amount_1',
+    'donation_suggestion_amount_2',
+    'donation_suggestion_amount_3',
+    'donation_suggestion_amount_4',
+    'donation_title',
+    'profile_button_text',
+    'profile_category',
+    'profile_name',
+    'profile_cover_image_url',
+  ]);
+}
+
+export async function down(_pgm: MigrationBuilder): Promise<void> {
+  console.log("not defining down migration for card_spaces' unused fields");
+}
```

Comment:
Is this okay to do? 😳

Reply:
Hmm... tbh I feel like it doesn't matter too much since these fields were never used, but I can see how it could be annoying for someone doing debugging, thinking about it a bit more. It isn't too much trouble, so I'll add the down migration.